server/sonar-server/src/main/java/org/sonar/server/user/index/UserResultSetIterator.java

```diff
@@ -57,38 +60,38 @@
 
   private final ListMultimap<String, String> organizationUuidsByLogins;
 
-  private UserResultSetIterator(PreparedStatement stmt, ListMultimap<String, String> organizationUuidsByLogins) throws SQLException {
+  private UserResultSetIterator(Collection<EsQueueDto> esQueueDtos, PreparedStatement stmt, ListMultimap<String, String> organizationUuidsByLogins) throws SQLException {
     super(stmt);
     this.organizationUuidsByLogins = organizationUuidsByLogins;
   }
 
-  static UserResultSetIterator create(DbClient dbClient, DbSession session, @Nullable List<String> logins) {
+  static UserResultSetIterator create(DbClient dbClient, DbSession session, @Nullable Collection<EsQueueDto> esQueueDtos) {
     try {
-      String sql = createSql(logins);
+      String sql = createSql(esQueueDtos);
       PreparedStatement stmt = dbClient.getMyBatis().newScrollingSelectStatement(session, sql);
-      setParameters(stmt, logins);
+      setParameters(stmt, esQueueDtos);
 
       ListMultimap<String, String> organizationUuidsByLogin = ArrayListMultimap.create();
-      if (logins == null) {
+      if (esQueueDtos == null) {
         dbClient.organizationMemberDao().selectAllForUserIndexing(session, organizationUuidsByLogin::put);
       } else {
-        dbClient.organizationMemberDao().selectForUserIndexing(session, logins, organizationUuidsByLogin::put);
+        dbClient.organizationMemberDao().selectForUserIndexing(session, esQueueDtos.stream().map(dto -> dto.getDocUuid()).collect(toList()), organizationUuidsByLogin::put);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)